iceoryx_posh/source/roudi/roudi_process.cpp

```diff
@@ -280,7 +280,6 @@ bool ProcessManager::isProcessAlive(const RouDiProcess& process) noexcept
         }
         return true;
```

Comment:
I would remove this return value and the whole else branch. Some tools complain if there is no return statement at the end of a method, even if all the branches have one